src/common/components/clusterConfiguration/DiscoveryTroubleshootingModal.tsx

```diff
@@ -14,9 +14,31 @@ import { useTranslation } from '../../hooks/use-translation-wrapper';
 import { Trans } from 'react-i18next';
 import { saveAs } from 'file-saver';
 import { CHANGE_ISO_PASSWORD_FILE_LINK } from '../../config/constants';
+import * as Sentry from '@sentry/browser';
 
 export const DiscoveryTroubleshootingModalContent = () => {
   const { t } = useTranslation();
+
+  const downloadIsoPasswordScript = async () => {
+    await fetch(CHANGE_ISO_PASSWORD_FILE_LINK)
+      .then((response) => {
+        response
+          .text()
+          .then((fileContent: string) => {
+            const blob = new Blob([fileContent], { type: 'text/plain;charset=utf-8' });
+            saveAs(blob, 'change-iso-password.sh');
+          })
+          .catch((err) => {
+            Sentry.captureException(err);
+            return {};
+          });
+      })
+      .catch((err) => {
+        Sentry.captureException(err);
+        return {};
+      });
+  };
```

Comment:
```suggestion
  const downloadIsoPasswordScript = async () => {
    try {
      const response = await fetch(CHANGE_ISO_PASSWORD_FILE_LINK);
      const fileContent = await response.text();
      const blob = new Blob([fileContent], { type: 'text/plain;charset=utf-8' });
      saveAs(blob, 'change-iso-password.sh');
    } catch (error) {
      Sentry.captureException(error);
    }
  };
```
BTW, why are you returning an empty object? Nobody is consuming the returned values from this function.